modules/extensions-support/src/main/java/org/mule/module/extension/internal/manager/ExtensionRegistry.java

```diff
@@ -82,7 +82,9 @@ boolean containsExtension(String name)
     }
 
     /**
-     * @param name the registration of the extension you want
+     * Returns a registered {@link Extension} of the given {@code name}
+     *
+     * @param name the name of the extension you're looking for
```

Comment:
name of the extension you're looking for > name of the extension that is being retrieved